docs/integrations/bitbucketServer/locations.md

```diff
@@ -0,0 +1,49 @@
+---
+id: locations
+title: Bitbucket Server Locations
+sidebar_label: Locations
+# prettier-ignore
+description: Integrating source code stored in Bitbucket Server into the Backstage catalog
+---
+
+The Bitbucket Server integration supports loading catalog entities from Bitbucket Server.
+Entities can be added to
+[static catalog configuration](../../features/software-catalog/configuration.md),
+or registered with the
+[catalog-import](https://github.com/backstage/backstage/tree/master/plugins/catalog-import)
+plugin.
+
+## Configuration
+
+```yaml
+integrations:
+  bitbucketServer:
+    - host: bitbucket.mycompany.com
+      apiBaseUrl: https://bitbucket.mycompany.com/rest/api/1.0
```

Comment:
**maybe** it makes sense to add this to the Basic Auth example, too. Just to not make the impression that it is needed for token-based but not for Basic Auth. Or remove it entirely as you have an example at the config option's documentation already.